lib/cli/Cargo.toml

```diff
@@ -35,11 +35,12 @@ wasmer-engine-universal = { version = "2.0.0", path = "../engine-universal", opt
 wasmer-engine-dylib = { version = "2.0.0", path = "../engine-dylib", optional = true }
 wasmer-engine-staticlib = { version = "2.0.0", path = "../engine-staticlib", optional = true }
 wasmer-vm = { version = "2.0.0", path = "../vm" }
-wasmer-wasi = { version = "2.0.0", path = "../wasi", default-features = false, optional = true }
+wasmer-wasi = { version = "2.0.0", path = "../wasi", default-features = false, features = ["host_fs"], optional = true }
```

Comment:
OK, so previously we were using the default features of `wasmer-wasi`. Now, `wasmer-wasi` has `host_fs` as the default feature, so that the previous behavior is kept, which is the correct way to do.

So… we don't need to update `wasmer-c-api` and `wasmer-cli` here. It's been fixed in https://github.com/wasmerio/wasmer/pull/2523.